Assets/Scripts/Weapons/BehaviourTypes/BulletMono.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BulletMono : MonoBehaviour
+{
+    float _damage;
+    public float Damage 
+    { 
+        get => _damage; 
+        set => _damage = value; 
+    }
+
+    float _bulletSpeed = 1f;
+    public float BulletSpeed {
+        get => _bulletSpeed;
+        set => _bulletSpeed = value;
+    }
+
+    float _bulletSpawnTime;
+
+    public static void create(Vector3 position, Vector3 direction, Dictionary<string, float> statsModifiers)
+    {
+        GameObject newObject = Instantiate(Resources.Load("Bullet")) as GameObject;
+        newObject.transform.position = position;
+        newObject.transform.forward = direction;
+
+        BulletMono bulletComponent = newObject.GetComponent<BulletMono>();
+        bulletComponent.Damage = statsModifiers["damage"];
```

Comment:
Shouldn't stats**Modifiers** be modifiers that change a base damage, rather than become the damage?

Reply:
Yup, I'll add base damage to the Bullet Mono